store/postgres/src/catalog.rs

```diff
@@ -214,7 +214,7 @@ pub fn account_like(conn: &PgConnection, site: &Site) -> Result<HashSet<String>,
         .select((ts::table_name, ts::is_account_like))
         .get_results::<(String, Option<bool>)>(conn)
         .optional()?
-        .unwrap_or(vec![])
+        .unwrap_or_default()
```

Comment:
I don't think this improves readability, `unwrap_or(vec![])` is shorter and documents the type.

Reply:
:+1: